### PR TITLE
project-business-site

### PR DESCRIPTION
**Pull request by Irina-web-dev**
During this project I have used learnings from the previous project. I feel quite good about the box-model approach and flexbox.
 
I don´t quite like how responsiveness works in this project. Maybe I can have feedback on that, what could have been done differently? With mobile first approach it was easier and I will try to stick with this approach for the next projects.

Looking forward receiving your code review.

### CODE REVIEW

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi Irina! I really like your design although some small responsiveness-bugs sneaked in. Please have a look at that before approval. Also, take a look at the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475) for this week to get more tips :) Have a nice weekend!

**Comment by Irina-web-dev** [comment]
> Hi Irina! I really like your design although some small responsiveness-bugs sneaked in. Please have a look at that before approval. Also, take a look at the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475) for this week to get more tips :) Have a nice weekend!

File: code/script.js
Comment (Irina-web-dev) [reply code/script.js]: Hi @HIPPIEKICK ,

Thank you for your code review and feedback.
I made the changes and I also changed the structure of styling to mobile first approach, it seemed much more convenient.

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**
